appserver/tests/pom.xml

```diff
@@ -50,17 +50,44 @@
     </parent>
 
     <groupId>fish.payara.server.internal.tests</groupId>
-    <artifactId>tests</artifactId>
+    <artifactId>payara-tests</artifactId>
```

Comment:
affects payara-samples module

Reply:
I think it does not, payara-samples inherit from another module, payara-aggregator. And it is not compilable, I wanted to fix it, but I had to postpone it, because there is much more issues ... but they are not related to this (but this can make them visible, because it integrates samples to the reactor).

Samples still use old group ids before Patrik's changes.

Reply:
So it is!. I assumed similarity :pensive: